packages/lib-classifier/src/components/Classifier/components/SubjectViewer/SubjectViewer.js

```diff
@@ -30,9 +37,14 @@ class SubjectViewer extends React.Component {
   }
 
   [asyncStates.success] () {
-    const { subject } = this.props
+    const { className, subject } = this.props
     const Viewer = getViewer(subject.viewer)
-    return <Viewer subject={subject} {...this.props} />
+    return (
```

Comment:
Having the SVG layer on always on top regardless of the viewer type is what breaks native HTML 5 video player controls, which is especially annoying on mobile devices. We've run into this problem on PFE with the video projects. I don't think we've had a project ask for SVG drawing annotations on video yet and something like that might be hard to implement for annotations. 

I think the text subject types that use the highlighter tool also don't need an SVG layer.

Reply:
Fair enough - I'll defer to the individual subject viewer on how it renders then.